### DISCUSSION TITLE
Tags das Issues

### DISCUSSION THREAD

**Discussion by LoriaLawrenceZ** [OZimbres/FIAP-Fase1#11] [Aberto]
<h1>Organização das Tags das Issues</h1>
<p>Eu estava pensando em deixar algumas tags de "aviso" (como "duplicata" ou "Ignora" por exemplo) totalmente em maiúsculo, pois serviriam de aviso pois afinam, avisam sobre a condição da issue, e não apenas serve como objeto de filtragem, o que acham (de deixar essas tags específicas) totalmente em maiúsculo, diferentemente das tags de tecnmologias ou gestão (organização, documentação etc)?</p>

<hr>

<details close>
    <summary>Menções</summary>
    @Guilherme-Avellar @LoriaLawrenceZ @HenriqueSaniW @ivan-maze @TheBaltarO
</details>

**Comment by TheBaltarO** [comment]
Meu toc foi mais forte que eu kk, mas faz sentido. 
Deixei maiúsculo como estava.

**Reply by LoriaLawrenceZ** [reply]
kkk não po, se tu achar mais válido pode deixar, mas eu queria fazer isso mais pra essas tags de "aviso" (Duplicada, pra mostrar q a issue tá duplicada, IGNORAR, pra ignorar tag e tals), não todas sabe
